src/models/groupModel.js

```diff
@@ -0,0 +1,31 @@
+import uuid from 'uuid/v1';
+import Sequelize, { DataTypes } from 'sequelize';
+
+import db from '../config/db';
+
+const permissions = ['READ', 'WRITE', 'DELETE', 'SHARE', 'UPLOAD_FILES'];
+
+const groupModelTableName = 'groups';
+const groupTableId = 'id';
+const GroupModel = db.define(groupModelTableName, {
+    [groupTableId]: {
```

Comment:
You're using dynamic object properties approach ( usage of "[]" brackets) for this config field - what about other fields like "name" and "permissions"? If there's no special reason for the "groupTableId" why you're only making its name dynamic - it might make sense to improve the overall consistency and make the rest of the fields (name and permission) also dynamic.

Reply:
Because I'm using table name and table id in another models to have relationship. Like in `userGroupModel`. What would you suggest ?

Reply:
Oh I see. Then it's ok!
Only suggestion that comes to mind for your approach - move these shared field names up to some common place at models level.